kubetest2-tf/hack/terraform_install.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# Copyright 2025 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TF_VERSION="v1.9.8"
+TERRAFORM_PROVIDER_IBM_VERSION="v1.73.0"
+TERRAFORM_PROVIDER_NULL_VERSION="v3.2.3"
+TF_PLUGIN_PATH=/root/.terraform.d/plugins/registry.terraform.io
+
+install_terraform_power(){
+    if [ -f "/go/bin/terraform" ]; then
+        echo "terraform already present"
+    else
+        cd /tmp
+        git clone -b $TF_VERSION --single-branch https://github.com/hashicorp/terraform.git
+        cd terraform
+        go install .
+    fi
+}
+
+install_terraform_x86(){
+    cd /tmp
+    if [ -f "/usr/local/bin/terraform" ]; then
+        echo "terraform already present"
+    else
+        curl -fsSL https://releases.hashicorp.com/terraform/${TF_VERSION:1}/terraform_${TF_VERSION:1}_linux_amd64.zip -o ./terraform.zip
+        unzip -o ./terraform.zip  >/dev/null 2>&1
+        rm -f ./terraform.zip
+        cp terraform /usr/local/bin/
+    fi
+}
+
+build_ibm_provider(){
+    if [[ ! -f "${TF_PLUGIN_PATH}/IBM-Cloud/ibm/${TERRAFORM_PROVIDER_IBM_VERSION}/linux_ppc64le/terraform-provider-ibm" || ! -f "${TF_PLUGIN_PATH}/hashicorp/ibm/${TERRAFORM_PROVIDER_IBM_VERSION}/linux_ppc64le/terraform-provider-ibm" ]]; then
+        cd /tmp
+        git clone -b ${TERRAFORM_PROVIDER_IBM_VERSION} --single-branch https://github.com/IBM-Cloud/terraform-provider-ibm
+        cd terraform-provider-ibm
+        go build .
+    fi
+}
+
+build_null_provider(){
+    if [ ! -f "${TF_PLUGIN_PATH}/hashicorp/null/${TERRAFORM_PROVIDER_NULL_VERSION}/linux_ppc64le/terraform-provider-null" ]; then
```

Comment:
move build and setup in one block to avoid unwanted checks.

Reply:
Fixed.